apps/log_retention_policy_enforcer/README.md

```diff
@@ -0,0 +1,3 @@
+# Delete Old ES Indices
```

Comment:
This readme looks incorrect

Reply:
Ah yes, missed this in the copy. One sec and I'll fix this.